test/labeling/model/test_wrapper.py

```diff
@@ -0,0 +1,54 @@
+import unittest
+
+import numpy as np
+from sklearn.model_selection import GridSearchCV
+
+from snorkel.labeling import SklearnLabelModel
+
+
+class LabelModelWrapperTest(unittest.TestCase):
+    def test_create_param_search_data(self) -> None:
+        L_train = np.array([[1, 1, 0], [-1, -1, 1], [0, 0, 1], [1, 1, 0]])
+        L_dev = np.array([[-1, 1, 1], [0, -1, 0]])
+        Y_dev = np.array([0, 1])
+
+        # combined L, Y
+        L_true = np.array(
+            [[1, 1, 0], [-1, -1, 1], [0, 0, 1], [1, 1, 0], [-1, 1, 1], [0, -1, 0]]
+        )
+        Y_true = np.array([-1, -1, -1, -1, 0, 1])
+
+        wrapper = SklearnLabelModel()
+        L, Y, cv_split = wrapper.create_param_search_data(L_train, L_dev, Y_dev)
+        np.testing.assert_array_equal(L, L_true)
+        np.testing.assert_array_equal(Y, Y_true)
+        np.testing.assert_array_equal(cv_split.get_n_splits(), 1)
+
+        L_dev = np.array([[-1, 1, 1], [0, -1, 0], [1, 1, 0]])
+        with self.assertRaisesRegex(ValueError, "Num. datapoints in Y_dev and L_dev"):
+            L, Y, cv_split = wrapper.create_param_search_data(L_train, L_dev, Y_dev)
+
+    def test_search(self):
+        L_train = np.array(
+            [[1, 1, 0, 0], [0, 1, 1, 0], [1, 0, 0, 1], [0, 1, 0, 1], [0, 0, 0, 1]]
+        )
+        L_dev = np.array([[1, 1, 1, 0], [0, 1, 1, 0], [1, 0, 0, 1], [0, 0, 1, 1]])
+        Y_dev = np.array([1, 1, 0, 1])
+
+        wrapper = SklearnLabelModel()
+        L, Y, cv_split = wrapper.create_param_search_data(L_train, L_dev, Y_dev)
+
+        param_grid = [{"n_epochs": [5, 100], "lr": [1e-10, 1e-2], "metric": ["f1"]}]
+
+        clf = GridSearchCV(wrapper, param_grid, cv=cv_split)
+        clf.fit(L, Y)
+
+        self.assertEqual(
+            clf.best_params_, {"lr": 0.01, "metric": "f1", "n_epochs": 100}
```

Comment:
cool so we expect this since all other settings should be drastically underfit?

Reply:
might want to set n_epochs to [1, 10] just to be safer? e.g. if there are changes to label model behavior?

Reply:
rest do underfit. and its grid search, so it'll either pick 5 epochs or 100 epochs (and the former would be wrong)

Reply:
it's grid searching over just the 4 options, so it should underfit at 5 and work at 100